src/Matcher/ArrayMatcher.php

```diff
@@ -203,6 +203,7 @@ private function getPropertyAccessor()
         }
 
         $accessorBuilder = PropertyAccess::createPropertyAccessorBuilder();
+        $accessorBuilder->enableExceptionOnInvalidIndex();
```

Comment:
Well in that case I guess you can just update dependency to ``2.5^`` in this PR (it will be tagged as 2.0 anyway). Recommended solution for versions ``1.0`` and ``1.1`` will be update to ``2.0`` for now, and maybe somebody will find a way to fix it also there.